<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(1, 48, 63);
border: 4px Solid ;
border-color: rgb(18, 150, 173);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(18, 150, 173);
border-bottom-left-radius:30px;
border-bottom-right-radius:30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 75 17pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>To-Do List</string>
   </property>
  </widget>
  <widget class="QPushButton" name="close_btn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>30</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 0);
border-radius:10px;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="taskListWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>531</width>
     <height>241</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(1, 48, 63);
border: 4px Solid ;
border-color: rgb(18, 150, 173);
color: rgb(255, 255, 255);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>110</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Delete Task</string>
   </property>
  </widget>
  <widget class="QPushButton" name="completeButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>180</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(6, 159, 42);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-radius:20px;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Complete Task</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>350</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 142, 177);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
border-bottom-right-radius:20px;
border-top-right-radius:20px;
</string>
   </property>
   <property name="text">
    <string>Add Task</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(1, 48, 63);
border: 4px Solid ;
border-color: rgb(18, 150, 173);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(1, 48, 63);
border: 4px Solid ;
border-color: rgb(18, 150, 173);
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
border-bottom-left-radius:10px;
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(18, 150, 173);
border-top-left-radius:10px;
border-top-right-radius:10px;
font: 75 13pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> Enter:</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_7</zorder>
  <zorder>label_2</zorder>
  <zorder>close_btn</zorder>
  <zorder>taskListWidget</zorder>
  <zorder>deleteButton</zorder>
  <zorder>completeButton</zorder>
  <zorder>addButton</zorder>
  <zorder>taskLineEdit</zorder>
  <zorder>label_8</zorder>
  <zorder>dateTimeEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
